GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/SpaceViewModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿using GalaxyZooTouchTable.Models;
+using PanoptesNetClient;
+using PanoptesNetClient.Models;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Threading.Tasks;
+
+namespace GalaxyZooTouchTable.ViewModels
+{
+    public class SpaceViewModel : INotifyPropertyChanged
+    {
+        public List<TableSubject> CurrentGalaxies { get; set; } = new List<TableSubject>();
+        public double RA { get; set; } = 250.2;
+        public double DEC { get; set; } = 35.1;
```

Comment:
Are these the hardcoded values you mentioned in the PR intro?

Reply:
Yes, these will change in the future, but they are hardcoded now because I'm working with a test subject set that contains subjects within these specific RA/DEC bounds.